vendor/bat-native-ads/src/bat/ads/internal/bundle/creative_daypart_info.h

```diff
@@ -13,6 +13,12 @@
 namespace ads {
 
 struct CreativeDaypartInfo final {
+  CreativeDaypartInfo();
```

Comment:
constructor/destructor for struct is not required.
could we do external comparison operators for CreativeDaypartInfo instead of making it as member.

`bool operator==(const CreativeDaypartInfo& a, const CreativeDaypartInfo& b)`